new-dti-website/src/app/page.tsx

```diff
@@ -95,7 +95,8 @@ const Home: React.FC = () => {
             <h1 className="text-white md:text-[40px] xs:text-[28px] z-10 font-medium lg:max-w-[442px] text-center">
               Building the Future of Tech @ Cornell
             </h1>
-            <div className="flex justify-center lg:hidden">
+            <div className="flex justify-center lg:hidden relative">
+              <div className="left-2/ absolute before:content-[''] before:absolute before:w-[500px] before:h-[270px] before:rounded-[128px] before:bg-[radial-gradient(50%_50%_at_50%_50%,_rgb(255_11_11_/_100%)_0%,_rgba(214,_61,_61,_0.70)_100%)] before:filter blur-[250px] before:left-2/4 before:-translate-x-1/2 before:translate-y-1/2" />
```

Comment:
Hi @Bookie0! This looks great! Just a few suggestions!

1. I think you have a typo here with left-2/! 
2. I believe before:content-[''] is redundant in this case! 
3. Also commas aren't necessary when defining rgba in tailwind! 
4. Added layout styles to the beginning

```suggestion
<div className="absolute before:absolute before:left-2/4 before:-translate-x-1/2 before:translate-y-1/2 before:w-[800px] before:h-[270px] before:rounded-[128px] before:bg-[radial-gradient(50%_50%_at_50%_50%,_rgb(255_11_11)_0%,_rgba(214_61_61_/_70%)_100%)] before:filter before:blur-[250px]" />
```

Reply:
sounds good thx!